models/PatientEntry.js

```diff
@@ -1,24 +1,52 @@
 const mongoose = require('mongoose');
+const Schema = mongoose.Schema;
+const PatientEntrySchema = new Schema({
 
-const PatientEntrySchema = new mongoose.Schema({  //schema
-    patientEmail: {
-        type: String,
-        required: true
-    },
-    form: {
-        type: [{date: Date, symptom1: String, symptom2: String, symptom3: String, symptom4: String, additionalNote: String, media: String}],
-        required: true,
+
+ patientFullName:
+     {
+        firstName: {type: String,rquired: true },
+        lastName: {type: String,rquired: true },
+      },
+
+ 
+  form: {
+       date: {type: Date,rquired: true },
+    symptom1:{type: Boolean },
+    symptom2: {type: Boolean },
+    symptom3: {type: Boolean },
+             
+    symptom4: {type: Boolean },
```

Comment:
It looks like symptoms in the model are set as booleans and in the front end they are imputed as strings.